renderer/index.js

```diff
@@ -48,12 +56,22 @@ function init () {
   })
   document.body.appendChild(vdomLoop.target)
 
+  // Calling update() updates the UI given the current state
+  // For performance, we don't want to do this too often. Hence, events that
+  // affect the UI can be v frequent--like torrent progress updates--call
+  // updateThrottled() instead, which calls update() at most once a second.
   updateThrottled = throttle(update, 1000)
+  setInterval(updateThrottled, 1000)
```

Comment:
This is actually a derp -- limiting `updateThrottled` to *at most once a second*, then calling it exactly once a second, is equivalent to removing `updateThrottled` entirely everywhere it's called and just saying

    setInterval(update, 1000)

@feross does that look right?